qemu/tests/block_hotplug_passthrough.py

```diff
@@ -0,0 +1,89 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_test
+from virttest import utils_disk
+from avocado.utils import process
+from provider.storage_benchmark import generate_instance
+from provider.block_devices_plug import BlockDevicesPlug
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Hotplug/unplug passthrough disk test:
+    1) Create passthrough disk with scsi_debug tool.
+    2) Start the guest.
+    3) Hotplug this passthrough disk.
+    4) Create partition on this disk and format it.
+    5) Do iozone/dd test on this disk.
+    6) Reboot the guest.
+    7) Unplug this passthrough disk.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def create_path_disk():
+        """Create a passthrough disk with scsi_debug """
+        create_cmd = params["create_command"]
+        process.system_output(create_cmd, timeout=300, shell=True, verbose=False)
+
+    def hotplug_path_disk(vm):
+        """Hotplug passthrough disk."""
+        error_context.context("Hotplug passthrough device", logging.info)
+        path_dev = process.getoutput("ls /dev/sd* | tail -n 1", shell=True)
```

Comment:
I thought the way to get new disk is to compare the set between before and after create_command since if failed to create the emulate scsi device, you will get the disk is not you wanna.

Reply:
ok, will try to update it, thanks~